package.json

```diff
@@ -50,7 +50,7 @@
     "react-storybook-addon-props-combinations": "^0.3.0",
     "react-tabs": "^0.8.2",
     "styled-components": "^1.4.3",
-    "tachyons-egghead": "^2.0.0"
+    "tachyons-egghead": "^2.3.1"
```

Comment:
Default background color is now $base, default link color is now $blue, default global markdown "code" now uses $green